### PR TITLE
Revive Wordcloud Extension

### LINKED ISSUE #161: Revive word-cloud extension
Hello,

Is it possible to bring word-cloud extension back?

### PR DESCRIPTION
**Pull request by ioachim-hub**
Resolves #161.

### CODE REVIEW

**Comment by mikhail-vl** [comment]
@ioachim-hub Anything wrong with the PR?

  I was about to include it in the v5 and release it.

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ grafana-cli plugins install volkovlabs-echarts-panel
 - Supports SVG and Canvas renderers.
 - Includes USA and World GeoJSON maps. Additional maps can be loaded dynamically.
 - Includes [Liquid Fill Chart](https://github.com/ecomfe/echarts-liquidfill), which is usually used to represent data in percentage.
+- Includes [WordCloud Chart](https://github.com/ecomfe/echarts-wordcloud), which is usually used to represent data in percentage.
```
Comment (nicolaerosia) [inline README.md:49]: fix the description

Comment (mikhail-vl) [reply README.md:49]: Thank you. I will update and tests are failing. I need to fix them as well.

**Comment by ioachim-hub** [comment]
@mikhail-vl I think there is no issue (or I can not see an issue) (I just saw that the tests are failing)

**Comment by mikhail-vl** [comment]
Works as expected.

Screenshot 2023-07-19 at 07 26 49

```
return {
  tooltip: {},
  series: [{
    type: 'wordCloud',
    gridSize: 2,
    sizeRange: [12, 100],
    shape: 'pentagon',
    drawOutOfBound: true,
    textStyle: {
      color: function () {
        return 'rgb(' + [
          Math.round(Math.random() * 160),
          Math.round(Math.random() * 160),
          Math.round(Math.random() * 160)
        ].join(',') + ')';
      }
    },
    emphasis: {
      textStyle: {
        shadowBlur: 10,
        shadowColor: '#333'
      }
    },
    data: [
      {
        name: 'Sam S Club',
        value: 10000,
        textStyle: {
          color: 'black'
        },
        emphasis: {
          textStyle: {
            color: 'red'
          }
        }
      },
      {
        name: 'Macys',
        value: 6181
      },
      {
        name: 'Amy Schumer',
        value: 4386
      },
      {
        name: 'Jurassic World',
        value: 4055
      },
      {
        name: 'Charter Communications',
        value: 2467
      },
      {
        name: 'Chick Fil A',
        value: 2244
      },
      {
        name: 'Planet Fitness',
        value: 1898
      },
      {
        name: 'Pitch Perfect',
        value: 1484
      },
      {
        name: 'Express',
        value: 1112
      },
      {
        name: 'Home',
        value: 965
      },
      {
        name: 'Johnny Depp',
        value: 847
      },
      {
        name: 'Lena Dunham',
        value: 582
      },
      {
        name: 'Lewis Hamilton',
        value: 555
      },
      {
        name: 'KXAN',
        value: 550
      },
      {
        name: 'Mary Ellen Mark',
        value: 462
      },
      {
        name: 'Farrah Abraham',
        value: 366
      },
      {
        name: 'Rita Ora',
        value: 360
      },
      {
        name: 'Serena Williams',
        value: 282
      },
      {
        name: 'NCAA baseball tournament',
        value: 273
      },
      {
        name: 'Point Break',
        value: 265
      }
    ]
  }]
};
```

**Comment by ioachim-hub** [comment]
I used it on my forked repo
Thank you, @mikhail-vl!!! ❤️

**Comment by mikhail-vl** [comment]
> I used it on my forked repo
> Thank you, @mikhail-vl!!! ❤️

@ioachim-hub What data source do you use it with?

**Comment by ioachim-hub** [comment]
> > I used it on my forked repo
> > Thank you, @mikhail-vl!!! heart
> 
> @ioachim-hub What data source do you use it with?

JSON datasource

**Review by mikhail-vl — APPROVED** [review]
Looks good!

**Pull request merged**